skbio/diversity/alpha/_base.py

```diff
@@ -295,6 +295,52 @@ def esty_ci(counts):
     return f1 / n - z * np.sqrt(W), f1 / n + z * np.sqrt(W)
 
 
+@experimental(as_of="0.4.0-dev")
+def faith_pd(counts, otu_ids, tree):
+    """ Compute Faith's phylogenetic diversity metric (PD)
+
+    Parameters
+    ----------
+    counts : 1-D array_like, int
+        Vector of counts.
+    otu_ids: list, np.array
+        Vector of OTU ids corresponding to tip names in ``tree``. Must be the
```

Comment:
Would it be more appropriate to use "observation" instead of OTU as it's more general?